awss3/s3_bucket.go

```diff
@@ -338,6 +330,20 @@ func (s *S3Bucket) putBucketPolicyWithRetries(
 	return err
 }
 
+func (s *S3Bucket) handleDeleteError(bucketName string, err error) error {
+	if awsErr, ok := err.(awserr.Error); ok {
```

Comment:
The more idiomatic way to do this is `errors.Is(err, target)`; do you know if the AWS errors implement `Is()` and can be compared in that way? (docs: https://pkg.go.dev/errors#Is)

Reply:
It looks like it is not supported: https://github.com/aws/aws-sdk-go/issues/3189